pkg/kn/commands/route/list_flags_test.go

```diff
@@ -0,0 +1,49 @@
+// Copyright © 2019 The Knative Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or im
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package route
+
+import (
+	"testing"
+
+	"github.com/knative/client/pkg/kn/commands"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+)
+
+func TestRoutListFlags(t *testing.T) {
```

Comment:
i dont think we need this test. it's too close to implementation so it doesnt add confidence that this object is correct. additionally route list flags object is used in the command test, so it gets tested there indirectly in a more resilient manner.

Reply:
I added this file's test as without this `pk/kn/commands/route/list_flags.go` was failing CI go test coverage threshold (must be > 50%).

should I be testing this in some other way?